go/base/static/js/src/routing/views.js

```diff
@@ -171,21 +171,49 @@
       this.diagram = options.diagram;
       this.states = this.diagram.states.members.get(this.collectionName);
       this.setElement(this.diagram.$('#' + this.id));
+      this.onDrag = this.onDrag.bind(this);
     },
 
-    repaint: function() { jsPlumb.repaintEverything(); },
+    onDrag: function() {
+      this.repaint();
+    },
 
-    render: function() {
-      this.states.each(function(s) { s.render(); });
+    repaintEl: function($el) {
+      // jsPlumb 1.7.5 doesn't seem to play nice with jquery ui's .sortable(),
```

Comment:
@SteveBarnett tbh, I'm not super confident on my understanding of why jsPlumb needs `.manage()`  :) what I do know is that we need to tell jsPlumb offsets explicitly when using `.repaint()`, and doing that causes jsPlumb to sometimes get unhappy. I'm a bit unclear on the unhappiness, but what I have gathered is that jsPlumb keeps a lookup of the elements it is managing (the elements that connectors and anchors attach to), and that this lookup appear to *sometimes* be empty. Calling `.manage()` puts the elements back in that lookup. I suspect the elements are being lost from that lookup either as a side effect of the `.repaint()`, or of some periodic task jsPlumb runs. If we simply ignored the unhappiness by doing something like `try { jsPlumb.repaint(...); } catch(e) {}`, jsPlumb still works as we want it to, which might suggest it is a bug in jsPlumb, but I'm not too sure. Ignoring errors is a terrible idea though, so calling `.manage()` seems a better idea.

The api docs, github issues and SO don't really point to a better solution (unless I'm not looking properly :)), which I'm guessing is because there aren't many people that have attempted to use jquery ui's `.sortable()` with newer jsPlumb, so this is really the best workaround I can think of without much knowledge of jsPlumb's codebase.

Hope that helps, happy to discuss this more on slack or something.